Scripts/Source/User/WorkshopParentScript.psc

```diff
@@ -1112,6 +1112,18 @@ Function RegisterCustomVendor(String aCustomVendorID, Formlist aVendorContainerL
 EndFunction
 
 
+function DumpCustomVendors()
```

Comment:
I found this to be useful for debugging, so I let this in.
I wasn't able to get the wsTrace function to work for me, so I used debug.trace. Maybe change it to something else, though.